planet/sync/client.py

```diff
@@ -34,7 +34,8 @@ class Planet:
           The session can be used to control the authentication method. Example:
 
           ```python
-          from planet import Auth, Session, Planet
+          from planet import Auth, Session
+          from planet.sync import Planet
 
           auth = Auth.from_key('examplekey')
```

Comment:
I know this is existing doc but maybe we should fix the example so folks don't get the idea they should hard-code their key into source code... e.g. 

`auth = Auth.from_key(get_key_from_encrypted_file())` 

or better, just note the Session can be provided allowing for whatever customization but defaults to standard behavior...